net/icmp/icmp_conn.c

```diff
@@ -109,13 +116,23 @@ FAR struct icmp_conn_s *icmp_alloc(void)
   ret = net_lockedwait(&g_free_sem);
   if (ret >= 0)
     {
+#ifdef CONFIG_NET_ALLOC_CONNS
+      if (dq_peek(&g_free_icmp_connections) == NULL)
+        {
+          conn = kmm_zalloc(sizeof(*conn) * CONFIG_NET_ICMP_NCONNS);
+          if (conn != NULL)
+            {
+              for (ret = 0; ret < CONFIG_NET_ICMP_NCONNS; ret++)
+                {
+                  dq_addlast(&conn[ret].node, &g_free_retcmp_connectretons);
```

Comment:
icmp/icmp_conn.c:127:48: error: `g_free_retcmp_connectretons` undeclared (first use in this function); did you mean `g_free_icmp_connections`?

Reply:
CI-Check doesn't have NET_ALLOC_CONNS enabled, otherwise it would have detected this. Could you enable it?

Reply:
emm... sorry for build break, I have fixed this issue on https://github.com/apache/incubator-nuttx/pull/5245 , please help to review, thank you!